web/src/widget.ts

```diff
@@ -68,3 +79,36 @@ export function frame_poll_message_content(): string {
         .filter(Boolean);
     return "/poll " + question + "\n" + options.join("\n");
 }
+
+export function todo_list_tasks_setup(): void {
+    const $todo_options_list = $("#add-todo-form .todo-options-list");
+    $todo_options_list.on("input", "input.todo-input", (event) => {
+        add_option_row(event, "TODO");
+    });
+    $todo_options_list.on("input", "input.todo-description-input", (event) => {
+        add_option_row(event, "TODO");
+    });
+    $todo_options_list.on("click", "button.delete-option", delete_option_row);
+
+    // setTimeout is needed to here to give time for simplebar to initialise
+    setup_sortable_list("#add-todo-form .todo-options-list");
+}
+
+export function frame_todo_message_content(): string {
+    const title = $<HTMLInputElement>("input#todo-title-input").val()?.toString().trim() ?? "";
+    const todo_str = title ? `/todo ${title}\n` : "";
```

Comment:
In case there is no title, the string is empty because the function `validate_input` used to insert the value `Task list`. However, I have removed `validate_input`, so now if the title is missing, the string "Task list" is entered through this line of code instead.